crates/compression/src/compressed_block_payload/v1.rs

```diff
@@ -29,12 +32,17 @@ pub struct CompressedBlockHeader {
     pub application: ApplicationHeader<Empty>,
     /// The consensus header.
     pub consensus: ConsensusHeader<Empty>,
-    // The block id.
+    /// The block id.
     pub block_id: BlockId,
+    /// The registry root before the block was produced.
```

Comment:
What is the point of having a registry root before the execution of the compressed block? It looks more logical to have root after the execution

Reply:
https://fuellabs.slack.com/archives/C04EWEF3XEZ/p1740050536289589?thread_ts=1740049590.995939&cid=C04EWEF3XEZ

Reply:
also, it's a stale comment. I checked the `registry_root` after the transaction was committed and it is the same as the value I use to create the `CompressedBlockPayloadV1`

Reply:
https://github.com/FuelLabs/fuel-core/commit/9656fd6239a796ddc694bbcba50a082da497c5ba#diff-d8d7fa8cb497931632674d904112a5bbea802757e71df7d92e06474a71463307R37